spring-cloud-kubernetes-client-discovery/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -1,3 +1,4 @@
 org.springframework.cloud.kubernetes.client.discovery.catalog.KubernetesCatalogWatchAutoConfiguration
-org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfiguration
```

Comment:
I dropped `KubernetesDiscoveryClientAutoConfiguration` and instead of it created its two inner classes (`KubernetesInformerDiscoveryAutoConfiguration` and `KubernetesDiscoveryClientHealthIndicatorAutoConfiguration`) as separate auto-configurations.